index.js

```diff
@@ -19,7 +19,7 @@ const moduleConfig = {
     node: true,
     ...(hasES2022Support ? { es2022: true } : { es6: true }),
   },
-  plugins: [ "eslint-plugin-n", "import" ],
+  plugins: [ "eslint-plugin-n", "import", "eslint-plugin-react" ],
```

Comment:
We have the react.js file that adds the plugin already (by extending @bonniernews/eslint-config/react)

Reply:
I have an issue with the tests failing if I don't add it here. Can I add it somewhere else?

Reply:
"react/jsx-curly-spacing": [ "error", { when: "never", children: { when: "always" } } ],  is in the wrong file, it should be in react-rules.js

Reply:
Ah, thanks!